### PR TITLE
Python docstrings

### PR DESCRIPTION
**Pull request by ljeub-pometry**
- Scala docstrings are available from python objects 
- Some automatic translation of scala to python syntax
- Running pytest in CI (more tests to be added)

### CODE REVIEW

**Comment by ljeub-pometry** [comment]
Method resolution order for python wrappers is currently broken, do not merge yet

**Comment by ljeub-pometry** [comment]
Ok, seems to work now

File: python/pyraphtory/poetry.lock [hunk 1]
```diff
@@ -0,0 +1,401 @@
+[[package]]
```
Comment (miratepuffin) [inline python/pyraphtory/poetry.lock:1]: This can be removed can't it?

File: python/pyraphtory/tests/test_pyraphtory.py [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 
 def test_version():
```
Comment (miratepuffin) [inline python/pyraphtory/tests/test_pyraphtory.py:4]: We have a version file now, can be read into here?

**Review by miratepuffin — CHANGES_REQUESTED** [review]

File: python/pyraphtory/poetry.lock (hunk 1 above)
Comment (ljeub-pometry) [reply python/pyraphtory/poetry.lock:1]: Poetry docs suggest to commit it so everyone ends up with the same packages, not sure it is that helpful.

File: python/pyraphtory/tests/test_pyraphtory.py (hunk 2 above)
Comment (ljeub-pometry) [reply python/pyraphtory/tests/test_pyraphtory.py:4]: That is a good idea, let me try

**Pull request merged**